fullscreen.bs

```diff
@@ -534,18 +582,18 @@ user.
  <code>top</code> is zero.
 </ul>
 
-<p>To <dfn>add</dfn> an <var>element</var> to a
+<p>To <dfn id=concept-top-layer-add>add</dfn> an <var>element</var> to a
 <var>top layer</var>, add, or move if already present, <var>element</var> on top of
 <var>top layer</var>.
 
-<p>To <dfn>remove</dfn> an <var>element</var> from a
+<p>To <dfn id=concept-top-layer-remove>remove</dfn> an <var>element</var> from a
 <var>top layer</var>, remove <var>element</var> from <var>top layer</var>.
 
 
 <h3 id=::backdrop-pseudo-element><code>::backdrop</code> pseudo-element</h3>
 
 <p>Each element in a <a>top layer</a> has a
-<dfn lt="css pe backdrop"><code>::backdrop</code></dfn> pseudo-element.
+<dfn id=css-pe-backdrop><code>::backdrop</code></dfn> pseudo-element.
```

Comment:
Adding it only adds `type="selector"` to the output, `data-dfn-type="selector"` is already generated automatically.